docs/source/distributed_systems.rst

```diff
@@ -26,19 +26,18 @@ Binding with distributed systems
 ================================
 
 Working with distributed systems is easy, you just basically need to make
-sure you use TCP transport and to specify the network interface where you
-want to bind.
+sure you use TCP transport and to specify the IP address that you want to bind
+to.
 
 This can be done when binding (note that we are using `127.0.0.1` but you
-may use any other interface available in your system):
+may use any other network interface IP address available in your system):
```

Comment:
You put "network interface IP address" here, but not in the paragraph above. I would rather put that in both or, at least, in the first paragraph. :wink: